infinite/index.ts

```diff
@@ -32,12 +31,11 @@ export const unstable_serialize = (getKey: InfiniteKeyLoader) => {
   return INFINITE_PREFIX + getFirstPageKey(getKey)
 }
 
-export const infinite = ((<Data, Error, Args extends Arguments>(
-  useSWRNext: SWRHook
-) => (
+export const infinite = ((<Data, Error>(useSWRNext: SWRHook) => (
   getKey: InfiniteKeyLoader,
-  fn: Fetcher<Data> | null,
-  config: typeof SWRConfig.default & SWRInfiniteConfiguration<Data, Error, Args>
+  fn: BareFetcher<Data> | null,
```

Comment:
Is Inifnite not be able to adopt Fetcher?

Reply:
This type is only used internally here.  It won't affect type inference